Flabbergasted-Flamingos/bangazon-api
# Rb readme update

# Description
Corrected a typo in computer model and added some doc strings and updated readme to include instructinos for seeder data

## Number of Fixes
3

## Related Ticket(s)
#9

## Problem to Solve
Describes how to install seeder data
Fixed typo in computers model
Added doc string to trainings


## Expected Behavior
Behaviour should remain the absolute same

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb-readme_update
```

 Please run migrations and runserver to make sure nothing is broken
